src/types/types.ts

```diff
@@ -1,3 +1,19 @@
-export type MachineLearningExampleType = {
-  name: string;
+export enum CertificateType {
+  PermissionOfPrinting = 'Permission of Printing',
+  OHSAS18001 = 'OHSAS 18001',
+}
+
+export type Certificate = {
+  id: number;
+  supplier: string;
+  certificateType: CertificateType;
+  validFrom: string;
+  validTo: string;
 };
+
+export enum TableHeaders {
+  SUPPLIER = 'Supplier',
```

Comment:
This is not correct ussage of enum in TS. You should not have for enum values something like translations, or pretty names for frontend. Expecially because that will not match with enums on backend. Displaying nice name on frontend for enum value should be handled in a different way.